libcaf_openssl/middleman_actor.cpp compile error on Windows

When compiling on Windows (VS2017 or VS2019) with OpenSSL enabled `middleman_actor.cpp` fails to compile with the following errors:
```
Microsoft (R) Build Engine version 16.4.0+e901037fe for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

  caf-generate-enum-strings.vcxproj -> C:\work\actor-framework\build-vs2019\bin\Debug\caf-generate-enum-strings.exe
  libcaf_core_static.vcxproj -> C:\work\actor-framework\build-vs2019\lib\Debug\caf_core_static.lib
  libcaf_io_static.vcxproj -> C:\work\actor-framework\build-vs2019\lib\Debug\caf_io_static.lib
  middleman_actor.cpp
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(103,1): warning C4005: 'AF_IPX': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(457): message : see previous definition of 'AF_IPX' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(147,1): warning C4005: 'AF_MAX': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(476): message : see previous definition of 'AF_MAX' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(185,1): warning C4005: 'SO_DONTLINGER': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(399): message : see previous definition of 'SO_DONTLINGER' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(238,25): error C2011: 'sockaddr': 'struct' type redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(1007): message : see declaration of 'sockaddr' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(440,5): error C2143: syntax error: missing '}' before 'constant' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(440,27): error C2059: syntax error: 'constant' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(494,1): error C2143: syntax error: missing ';' before '}' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(494,10): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(494,20): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(575,1): warning C4005: 'IN_CLASSA': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(284): message : see previous definition of 'IN_CLASSA' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(581,1): warning C4005: 'IN_CLASSB': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(290): message : see previous definition of 'IN_CLASSB' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(587,1): warning C4005: 'IN_CLASSC': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(296): message : see previous definition of 'IN_CLASSC' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(598,1): warning C4005: 'INADDR_ANY': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(301): message : see previous definition of 'INADDR_ANY' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(600,1): warning C4005: 'INADDR_BROADCAST': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(303): message : see previous definition of 'INADDR_BROADCAST' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(636,28): error C2011: 'sockaddr_in': 'struct' type redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(1011): message : see declaration of 'sockaddr_in' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(136,23): error C2011: 'fd_set': 'struct' type redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(1019): message : see declaration of 'fd_set' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(156,1): warning C4005: 'FD_CLR': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(94): message : see previous definition of 'FD_CLR' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(171,1): warning C4005: 'FD_SET': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(99): message : see previous definition of 'FD_SET' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(180,16): error C2011: 'timeval': 'struct' type redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(1035): message : see declaration of 'timeval' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(236,17): error C2011: 'hostent': 'struct' type redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(1023): message : see declaration of 'hostent' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(249,16): error C2011: 'netent': 'struct' type redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(177): message : see declaration of 'netent' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(256,17): error C2011: 'servent': 'struct' type redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(1027): message : see declaration of 'servent' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(268,18): error C2011: 'protoent': 'struct' type redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(1031): message : see declaration of 'protoent' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(364,24): error C2011: 'WSAData': 'struct' type redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(319): message : see declaration of 'WSAData' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(462,18): error C2011: 'sockproto': 'struct' type redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(491): message : see declaration of 'sockproto' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(504,16): error C2011: 'linger': 'struct' type redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(1015): message : see declaration of 'linger' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(517,1): warning C4005: 'SOMAXCONN': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(541): message : see previous definition of 'SOMAXCONN' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(552,1): warning C4005: 'FD_READ': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(559): message : see previous definition of 'FD_READ' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(555,1): warning C4005: 'FD_WRITE': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(560): message : see previous definition of 'FD_WRITE' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(558,1): warning C4005: 'FD_OOB': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(561): message : see previous definition of 'FD_OOB' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(561,1): warning C4005: 'FD_ACCEPT': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(562): message : see previous definition of 'FD_ACCEPT' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(564,1): warning C4005: 'FD_CONNECT': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(563): message : see previous definition of 'FD_CONNECT' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(567,1): warning C4005: 'FD_CLOSE': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(564): message : see previous definition of 'FD_CLOSE' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1625,1): error C2375: 'accept': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(739): message : see declaration of 'accept' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1647,1): error C2375: 'bind': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(744): message : see declaration of 'bind' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1668,1): error C2375: 'closesocket': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(749): message : see declaration of 'closesocket' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1685,1): error C2375: 'connect': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(751): message : see declaration of 'connect' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1706,1): error C2375: 'ioctlsocket': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(756): message : see declaration of 'ioctlsocket' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1729,1): error C2375: 'getpeername': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(761): message : see declaration of 'getpeername' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1750,1): error C2375: 'getsockname': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(766): message : see declaration of 'getsockname' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1771,1): error C2375: 'getsockopt': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(771): message : see declaration of 'getsockopt' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1796,1): error C2375: 'htonl': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(778): message : see declaration of 'htonl' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1813,1): error C2375: 'htons': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(780): message : see declaration of 'htons' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1831,1): error C2375: 'inet_addr': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(782): message : see declaration of 'inet_addr' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1849,1): error C2375: 'inet_ntoa': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(784): message : see declaration of 'inet_ntoa' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1949,1): error C2375: 'listen': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(786): message : see declaration of 'listen' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1968,1): error C2375: 'ntohl': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(790): message : see declaration of 'ntohl' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1985,1): error C2375: 'ntohs': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(792): message : see declaration of 'ntohs' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2002,1): error C2375: 'recv': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(794): message : see declaration of 'recv' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2025,1): error C2375: 'recvfrom': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(800): message : see declaration of 'recvfrom' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2052,1): error C2375: 'select': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(808): message : see declaration of 'select' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2077,1): error C2375: 'send': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(815): message : see declaration of 'send' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2100,1): error C2375: 'sendto': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(821): message : see declaration of 'sendto' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2127,1): error C2375: 'setsockopt': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(829): message : see declaration of 'setsockopt' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2152,1): error C2375: 'shutdown': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(836): message : see declaration of 'shutdown' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2172,1): error C2375: 'socket': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(840): message : see declaration of 'socket' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2197,1): error C2375: 'gethostbyaddr': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(847): message : see declaration of 'gethostbyaddr' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2219,1): error C2375: 'gethostbyname': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(852): message : see declaration of 'gethostbyname' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2236,1): error C2375: 'gethostname': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(854): message : see declaration of 'gethostname' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2276,1): error C2375: 'getservbyport': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(858): message : see declaration of 'getservbyport' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2295,1): error C2375: 'getservbyname': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(862): message : see declaration of 'getservbyname' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2314,1): error C2375: 'getprotobynumber': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(866): message : see declaration of 'getprotobynumber' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2331,1): error C2375: 'getprotobyname': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(868): message : see declaration of 'getprotobyname' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2351,1): error C2375: 'WSAStartup': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(872): message : see declaration of 'WSAStartup' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2371,1): error C2375: 'WSACleanup': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(876): message : see declaration of 'WSACleanup' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2388,1): error C2375: 'WSASetLastError': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(878): message : see declaration of 'WSASetLastError' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2405,1): error C2375: 'WSAGetLastError': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(880): message : see declaration of 'WSAGetLastError' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2426,1): error C2375: 'WSAIsBlocking': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(882): message : see declaration of 'WSAIsBlocking' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2444,1): error C2375: 'WSAUnhookBlockingHook': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(884): message : see declaration of 'WSAUnhookBlockingHook' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2462,1): error C2375: 'WSASetBlockingHook': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(886): message : see declaration of 'WSASetBlockingHook' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2480,1): error C2375: 'WSACancelBlockingCall': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(888): message : see declaration of 'WSACancelBlockingCall' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2498,1): error C2375: 'WSAAsyncGetServByName': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(890): message : see declaration of 'WSAAsyncGetServByName' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2526,1): error C2375: 'WSAAsyncGetServByPort': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(898): message : see declaration of 'WSAAsyncGetServByPort' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2554,1): error C2375: 'WSAAsyncGetProtoByName': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(906): message : see declaration of 'WSAAsyncGetProtoByName' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2580,1): error C2375: 'WSAAsyncGetProtoByNumber': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(913): message : see declaration of 'WSAAsyncGetProtoByNumber' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2606,1): error C2375: 'WSAAsyncGetHostByName': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(920): message : see declaration of 'WSAAsyncGetHostByName' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2632,1): error C2375: 'WSAAsyncGetHostByAddr': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(927): message : see declaration of 'WSAAsyncGetHostByAddr' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2662,1): error C2375: 'WSACancelAsyncRequest': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(936): message : see declaration of 'WSACancelAsyncRequest' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2680,1): error C2375: 'WSAAsyncSelect': redefinition; different linkage [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(938): message : see declaration of 'WSAAsyncSelect' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(4206,1): error C2059: syntax error: '}' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(4206,1): error C2143: syntax error: missing ';' before '}' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2ipdef.h(42,12): error C2143: syntax error: missing ';' before '{' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2ipdef.h(42,12): error C2447: '{': missing function header (old-style formal list?) [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2ipdef.h(135,1): warning C4005: 'IP_TOS': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(349): message : see previous definition of 'IP_TOS' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2ipdef.h(136,1): warning C4005: 'IP_TTL': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(348): message : see previous definition of 'IP_TTL' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2ipdef.h(137,1): warning C4005: 'IP_MULTICAST_IF': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(343): message : see previous definition of 'IP_MULTICAST_IF' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2ipdef.h(138,1): warning C4005: 'IP_MULTICAST_TTL': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(344): message : see previous definition of 'IP_MULTICAST_TTL' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2ipdef.h(139,1): warning C4005: 'IP_MULTICAST_LOOP': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(345): message : see previous definition of 'IP_MULTICAST_LOOP' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2ipdef.h(140,1): warning C4005: 'IP_ADD_MEMBERSHIP': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(346): message : see previous definition of 'IP_ADD_MEMBERSHIP' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2ipdef.h(141,1): warning C4005: 'IP_DROP_MEMBERSHIP': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(347): message : see previous definition of 'IP_DROP_MEMBERSHIP' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\shared\ws2ipdef.h(142,1): warning C4005: 'IP_DONTFRAGMENT': macro redefinition [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(350): message : see previous definition of 'IP_DONTFRAGMENT' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(733,30): error C2061: syntax error: identifier 'MULTICAST_MODE_TYPE' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(740,5): error C2065: 'PIP_MSFILTER': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(740,18): error C2146: syntax error: missing ';' before identifier 'Filter' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(740,18): error C2065: 'Filter': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(742,9): error C2065: 'SourceCount': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(743,40): error C2065: 'Filter': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(743,60): error C2065: 'SourceList': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(744,9): error C2065: 'WSASetLastError': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(748,12): error C2065: 'IP_MSFILTER': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(748,12): error C2065: 'SourceCount': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(749,5): error C2065: 'Filter': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(749,15): error C2065: 'PIP_MSFILTER': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(749,29): error C2146: syntax error: missing ';' before identifier 'HeapAlloc' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(750,9): error C2065: 'Filter': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(751,9): error C2065: 'WSASetLastError': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(755,5): error C2065: 'Filter': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(756,5): error C2065: 'Filter': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(757,5): error C2065: 'Filter': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(757,26): error C2065: 'FilterMode': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(758,5): error C2065: 'Filter': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(758,27): error C2065: 'SourceCount': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(759,9): error C2065: 'SourceCount': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(760,1): error C2065: 'Filter': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(760,1): error C2065: 'SourceList': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(760,1): error C2065: 'SourceCount': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(760,1): error C2660: 'memcpy': function does not take 2 arguments [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include\vcruntime_string.h(40,15): message : see declaration of 'memcpy' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(764,47): error C2065: 'Filter': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(767,35): error C2065: 'Filter': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(779,31): error C2061: syntax error: identifier 'MULTICAST_MODE_TYPE' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(786,5): error C2065: 'PIP_MSFILTER': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(786,18): error C2146: syntax error: missing ';' before identifier 'Filter' [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(786,18): error C2065: 'Filter': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(788,10): error C2065: 'SourceCount': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(789,40): error C2065: 'Filter': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(789,60): error C2065: 'SourceList': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(790,9): error C2065: 'WSASetLastError': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(794,12): error C2065: 'IP_MSFILTER': undeclared identifier [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\um\ws2tcpip.h(794,12): fatal error C1003: error count exceeds 100; stopping compilation [C:\work\actor-framework\build-vs2019\libcaf_openssl\libcaf_openssl_static.vcxproj]
```

The following patch fixes the problem:
```
Index: libcaf_openssl/src/openssl/middleman_actor.cpp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- libcaf_openssl/src/openssl/middleman_actor.cpp	(revision b05a4b7b2a6f59b0508db84ea68e5ada2224f904)
+++ libcaf_openssl/src/openssl/middleman_actor.cpp	(date 1577046889132)
@@ -18,6 +18,14 @@
 
 #include "caf/io/middleman_actor.hpp"
 
+#ifdef CAF_WINDOWS
+#  include <winsock2.h>
+#  include <ws2tcpip.h> // socket_size_type, etc. (MSVC20xx)
+#else
+#  include <sys/socket.h>
+#  include <sys/types.h>
+#endif
+
 #include <stdexcept>
 #include <tuple>
 #include <utility>
@@ -42,14 +50,6 @@
 
 #include "caf/openssl/session.hpp"
 
-#ifdef CAF_WINDOWS
-#  include <winsock2.h>
-#  include <ws2tcpip.h> // socket_size_type, etc. (MSVC20xx)
-#else
-#  include <sys/socket.h>
-#  include <sys/types.h>
-#endif
-
 namespace caf {
 namespace openssl {

 
```

--- a/libcaf_openssl/src/openssl/middleman_actor.cpp
+++ b/libcaf_openssl/src/openssl/middleman_actor.cpp
@@ -18,6 +18,14 @@
 
 #include "caf/io/middleman_actor.hpp"
 
+#ifdef CAF_WINDOWS
+#  include <winsock2.h>
+#  include <ws2tcpip.h> // socket_size_type, etc. (MSVC20xx)
+#else
+#  include <sys/socket.h>
+#  include <sys/types.h>
+#endif
+
 #include <stdexcept>
 #include <tuple>
 #include <utility>
@@ -41,14 +49,6 @@
 #include "caf/io/network/stream_impl.hpp"
 
 #include "caf/openssl/session.hpp"
-
-#ifdef CAF_WINDOWS
-#  include <winsock2.h>
-#  include <ws2tcpip.h> // socket_size_type, etc. (MSVC20xx)
-#else
-#  include <sys/socket.h>
-#  include <sys/types.h>
-#endif
 
 namespace caf::openssl {
 

